src/v/pandaproxy/rest/handlers.cc

```diff
@@ -66,11 +66,6 @@ namespace {
 
 using server = ctx_server<proxy>;
 
-ss::shard_id consumer_shard(const kafka::group_id& g_id) {
```

Comment:
Oh, I see. Can you squash this with the previous commit so that every commit compiles?
```
error: unused variable 'group_shard' [-Werror,-Wunused-variable
```

Reply:
nice catch